src/components/Tabs/_tests_/Tab.test.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { Tabs, Tab } from "..";
+import { TabLabel } from "../TabLabel";
+
+describe('<Tab/>', () => {
+    test('renders without crushing', () => {
+        render(<Tab label="Steps"/>);
+    });
+    test('props work', () => {
+        const label = "Calls"
+        const testLabel = "NewCalls"
+        const { getByTestId } = render(<Tab label={label} info={testLabel}></Tab> );
+        const elem = getByTestId('label')
+        expect(getByTestId('label').textContent).toBe(testLabel);
+        
+
+    });
+});
+
+describe('<TabLabel/>', () => {
+    test('renders without crushing', () => {
+        let activeTab = false;
+        let label = "";
+        const clickTabItem = jest.fn(() => null);
+        render(<TabLabel            
+            active={!activeTab}
+            label={label}
+            clickTabItem={clickTabItem}
+          />);
+    });
+    test('clickTab', () => {
+        let clicked = false;
+        let activeTab = false;
+        const testLabel = "Workshops"
+
+        const { getByTestId } = render(
+        <TabLabel active={!activeTab}
+                  label={testLabel} 
+                  clickTabItem={() =>clicked = true}/>);
+        const tabList = getByTestId('labels');
+        fireEvent.click(tabList);
+        expect(clicked).toBe(true);
+
+    });
```

Comment:
this is a nice test case! I like your creative solution as well, but there is a more recommened way of testing whether a callback is called or not 
What you should do is to create a function called something like `mockClickTab` and declare it as
`const mockClickTab = jest.fn(() => null)`. You can then use it as 
```jsx
const { getByTestId } = render(
        <TabLabel active={!activeTab}
                  label={testLabel} 
                  clickTabItem={mockClickTab}
         />);
```

This way, you will have a jest mock function, on which you will be able to assertions through the `expect` function. Some of those assertions are `toHaveBeenCalled` and `toHaveBeenCalledWith`. The former tests that a function "has been called", whilst the latter takes any amount of arguments and it expects the function to have been called with the particularly given param. So for exmple if you expect that `mockClickTabItem`  should be called with a string that is equal to the label, you can then use an assertion like

```
expect(mockClickTab).tohaveBeenCalledWith(testLabel)
```